XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1472,6 +1472,29 @@
       "ImagePath": "",
       "Description": "",
       "Items": [
+        {
+          "UniqueId": "AnimatedVisualPlayer",
+          "Title": "AnimatedVisualPlayer",
+          "Subtitle": "The AnimatedVisualPlayer allows playback of animated Windows.UI.Composition.Visual trees.",
+          "ImagePath": "Assets/Animations.png",
+          "Description": "The AnimatedVisualPlayer allows playback of animated Windows.UI.Composition.Visual trees.",
+          "Content": "<p>Look at the <i>AnimatedVisualPlayerPage.xaml</i> and <i>AnimatedVisualPlayerPage.xaml.cs</i> files in Visual Studio to see the full code for this page.</p>",
+          "IsNew": true,
+          "Docs": [
+            {
+              "Title": "AnimatedVisualPlayer API",
+              "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedvisualplayer"
+            },
+            {
+              "Title": "Sample application with walkthrough of scenarios",
+              "Uri": "ms-windows-store://pdp/?productid=9N3J5TG8FF7F"
+            },
+            {
+              "Title": "Documentation and Tutorials",
```

Comment:
```suggestion
              "Title": "Guidelines and Tutorials",
```

Reply:
This link appears under the "Documentation" section on the page so we should be crisp about what kind of documentation it is. I suggested "Guidelines" but I'm open to other ideas. I just don't think "Documentation" under "Documentation" looks right.

Reply:
Agreed. Any chance you can remove the "Related Controls" list to the right as well?